src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -87,24 +87,28 @@ public bool MatchesPattern(string text, string pattern, CultureInfo culture)
             var patternMatcher = GetPatternMatcher(pattern, culture, includeMatchSpans);
             var match = patternMatcher.GetFirstMatch(completionItemText);
 
-            if (match != null)
+            if (culture.Equals(EnUSCultureInfo))
```

Comment:
Does this mean we'll now being doing matching twice for English that isn't en-US like en-GB? Is that a perf problem?